src/test/java/org/magicdgs/readtools/utils/fastq/FastqReadNameEncodingUnitTest.java

```diff
@@ -67,6 +76,11 @@
                         "@HWUSI-EAS100R:6:73:941:1973#ATCGA comment",
                         "@HWUSI-EAS100R:6:73:941:1973#ATCGA", false, false, false,
                         new String[] {"ATCGA"}},
+                {FastqReadNameEncoding.ILLUMINA,
+                        "@HWUSI-EAS100R:6:73:941:1973#ACTG/2 comment",
```

Comment:
Probably we should also add another test to check how something like "@readName comment#ACTG/2" works...